collector/controller.go

```diff
@@ -64,10 +64,6 @@ func NewManager(cfg *rest.Config, options ctrl.Options) (ctrl.Manager, error) {
 	if err := pipelinev1.AddToScheme(options.Scheme); err != nil {
 		return nil, err
 	}
-	options.NewCache = cache.BuilderWithOptions(cache.Options{
```

Comment:
the code was actually initializing `options.NewCache` twice in this method ... removed the first instance here